frontend/src/stores/accounts.ts

```diff
@@ -2,67 +2,200 @@ import { defineStore } from 'pinia';
 import { computed, ref } from 'vue';
 import type { Address } from '@/types';
 import { createAccount, generatePrivateKey } from 'genlayer-js';
-import type { Account } from 'genlayer-js/types';
 import { useShortAddress } from '@/hooks';
 
+export interface AccountInfo {
+  type: 'local' | 'metamask';
+  address: Address;
+  privateKey?: Address; // Only for local accounts
+}
```

Comment:
using `type` and `privateKey` nullability is redundant. I think the cleanest approach would be to have 2 interfaces, one called `LocalAccountInfo` with `address` and `privateKey`, and another `NonLocalAccountInfo` with only the `Address`. Then, using `LocalAccountInfo | NonLocalAccountInfo` we could represent this type

Reply:
This should be fixed now